src/main/java/com/sprint/mission/discodeit/entity/User.java

```diff
@@ -0,0 +1,47 @@
+package com.sprint.mission.discodeit.entity;
+
+import java.text.SimpleDateFormat;
+import java.util.UUID;
+
+public class User {
+    private UUID id;
+    private Long createdAt;
+    private Long updatedAt;
+
+    private String name;
+
+    public User(String name){
+        this.id = UUID.randomUUID();
+        this.createdAt = System.currentTimeMillis();
```

Comment:
생성시간을 초기화할 때 수정시간도 같이 초기화해주는게 자연스러울 것 같아요.  
```suggestion
        this.createdAt = System.currentTimeMillis();
        this.updatedAt = this.createdAt;
```